Examples/Tests/scraping/inputs_rz

```diff
@@ -41,7 +41,13 @@ electron.momentum_function_uy(x,y,z) = "if(x*x+y*y>0.0, -1.0*y/sqrt(x*x+y*y), 0.
 electron.momentum_function_uz(x,y,z) = "0"
 electron.do_not_deposit = 1
 
-diagnostics.diags_names = diag1
+diagnostics.diags_names = diag1 diag2
+
 diag1.intervals = 1
 diag1.diag_type = Full
 diag1.fields_to_plot = Ex
+
+diag2.intervals = 1
```

Comment:
To be technically clean for filtering tests, please add the compile flags here for openPMD:
https://github.com/ECP-WarpX/WarpX/blob/development/Regression/WarpX-tests.ini#L3351-L3352